x/epochstorage/client/cli/query_stake_storage_test.go

```diff
@@ -1,162 +1,149 @@
 package cli_test
 
 import (
-	"fmt"
 	"strconv"
 	"testing"
-
-	tmcli "github.com/cometbft/cometbft/libs/cli"
-	"github.com/cosmos/cosmos-sdk/client/flags"
-	clitestutil "github.com/cosmos/cosmos-sdk/testutil/cli"
-	"github.com/stretchr/testify/require"
-	"google.golang.org/grpc/codes"
-	"google.golang.org/grpc/status"
-
-	"github.com/lavanet/lava/testutil/network"
-	"github.com/lavanet/lava/testutil/nullify"
-	"github.com/lavanet/lava/x/epochstorage/client/cli"
-	"github.com/lavanet/lava/x/epochstorage/types"
 )
 
 // Prevent strconv unused error
 var _ = strconv.IntSize
 
-func networkWithStakeStorageObjects(t *testing.T, n int) (*network.Network, []types.StakeStorage) {
-	t.Helper()
-	cfg := network.DefaultConfig()
-	state := types.GenesisState{}
-	require.NoError(t, cfg.Codec.UnmarshalJSON(cfg.GenesisState[types.ModuleName], &state))
+// func networkWithStakeStorageObjects(t *testing.T, n int) (*network.Network, []types.StakeStorage) {
```

Comment:
delete the file

Reply:
done. see https://github.com/lavanet/lava/commit/043254df8a22e4d02246b902208872ab8b3e1e20